xbmc/LangInfo.h

```diff
@@ -74,9 +74,16 @@ class CLangInfo : public ISettingCallback, public ISettingsHandler
   const std::string& GetLanguageCode() const { return m_languageCodeGeneral; }
 
   /*!
-   \brief Returns the given language's name in English
+   * \brief Convert an english language name to locale,
+   *        by searching in the installed language addons.
+   * \param langName [OPT] The english language name
+   */
+  std::string ConvertEnglishNameToLocale(const std::string& langName) const;
```

Comment:
As far as I see, this does not need to be a member function. So, make it ‘static’, remove ‘const’.

Reply:
done